packages/opentelemetry-exporter-collector/src/platform/browser/CollectorExporterBrowserBase.ts

```diff
@@ -53,6 +56,10 @@ export abstract class CollectorExporterBrowserBase<
       );
     } else {
       this._headers = {};
+      this._blobPropertBag = {
```

Comment:
This seems like extreme overkill and doesn't actually solve the original issue properly. See additional comment below